elasticdl/python/elasticdl/image_builder.py

```diff
@@ -154,14 +154,14 @@ def _create_dockerfile(
 RUN pip install -r /elasticdl/requirements.txt \
   --extra-index-url="${EXTRA_PYPI_INDEX}"
 RUN make -f /elasticdl/Makefile
-COPY {MODEL_ZOO} /model_zoo/{MODEL_ZOO}
+COPY {MODEL_ZOO} /{MODEL_ZOO}
```

Comment:
I think the correct way is that in image_builder, we copy `model_def` into `/model_zoo`. 
For example, on the client, `cifar10_functional_api` is in `some_dir`. The user set `model_zoo=some_dir`.
After image_builder, `cifar10_functional_api` is under `/model_zoo` in the docker image.

Then in master pod argument, set `model_zoo=/model_zoo`.

Reply:
If `cifar10_functional_api` is in `some_dir`, the user will write 
`from some_dir.cifar10_function_api.some_module import some_class` to import `some_class` from `some_module.py`. 
If `cifar10_functional_api` is under `/model_zoo` in the docker image, the import error will happen and the corret import code in the image is
`from model_zoo.cifar10_function_api.some_module import some_class`

Reply:
ok, then I assume the user will use the root directory of `some_dir` as python directory.